.drone.star

```diff
@@ -41,6 +41,8 @@ DEFAULT_PHP_VERSION = "7.4"
 DEFAULT_NODEJS_VERSION = "14"
 
 dirs = {
+    "base": "/drone/src",
+    "web": "/drone/src/webTestRunner",
```

Comment:
There are quite a few hard-coded mentions of "webTestRunner".
Maybe we can have:
```suggestion
    "web_test_runner": "webTestRunner",
```
and use it like "core" and "testing_app" that are below - relative to "base". Then we can construct the base+web_test_runner path when needed, or just substitute dirs["web_test_runner"] when just that directory name is needed in a string.